forte/sparse_ci/ci_reference.cc

```diff
@@ -619,10 +652,28 @@ void CI_Reference::build_gas_single(std::vector<Determinant>& ref_space) {
 
     std::vector<std::vector<size_t>> rel_gas_mos;     // relative indices within active
     std::vector<std::vector<double>> rel_gas_eps(12); // ngas of SCF orbital energies
+    std::vector<std::vector<size_t>> gas_eps_idx(
+        12); // The index of orbitals in ascending order of orbital energies in each gas
     std::map<int, int> gas_nonzero_to_full;
     auto epsilon_a = scf_info_->epsilon_a();
     auto epsilon_b = scf_info_->epsilon_b();
 
+    // Find the sorted_indexes of a vector
+    auto sort_indexes = [&](const std::vector<double>& v) {
```

Comment:
Shall we put this function to helper because it may be a general interest for others. Shall we also rename this to `argsort` following numpy naming?

Reply:
I changed name of this function to argsort.